gtd-timer/Controllers/LogInController.cs

```diff
@@ -24,5 +27,19 @@ public IActionResult Login([FromBody] LoginDTO model)
 
             return Ok(token);
         }
+
+        [HttpPost]
+        public IActionResult GoogleLogin([FromBody] SocialAuthDTO accessToken)
+        {
+            var token = socialAuthService.CreateToken(accessToken, SocialAuthProvider.Google);
```

Comment:
Because I created only one service for this authentication. If create two different services, they will have a similar code, that is why I pass the second parament that shows type provider.
This service find a user in database and if don't exist add it user to database and return created jwt token.

Reply:
don't like this way of differentiating between google / facebook, but good enough for now. let's sync about it on our next meetup

Reply:
Ok